hasher-matcher-actioner/hmalib/lambdas/api/submit.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import bottle
+import boto3
+import base64
+
+from dataclasses import dataclass, asdict
+from mypy_boto3_dynamodb.service_resource import Table
+import typing as t
+
+from hmalib.lambdas.api.middleware import jsoninator, JSONifiable, DictParseable
+from hmalib.common.logging import get_logger
+
+logger = get_logger(__name__)
+s3_client = boto3.client("s3")
+dynamodb = boto3.resource("dynamodb")
+
+
+@dataclass
+class SubmitContentRequestBody(DictParseable):
+    submission_type: str  # TODO Enum
+    content_id: str
+    content_type: str  # TODO Enum
+    content_ref: t.Union[str, bytes]
+    metadata: t.Optional[t.List]
+
+    @classmethod
+    def from_dict(cls, d):
+        # ToDo Cleaner error handling
+        return cls(
+            d["submission_type"],
+            d["content_id"],
+            d["content_type"],
+            d["content_ref"],
+            d["metadata"],
+        )
+
+
+@dataclass
+class SubmitContentResponse(JSONifiable):
+    content_id_submitted: str
```

Comment:
can we change this to content_id? I think in most APIs I've seen you keep a name consistent if it represents the same thing.